Neos.Flow/Classes/Mvc/ActionResponse.php

```diff
@@ -153,32 +151,79 @@ public function deleteCookie(string $cookieName): void
      * @param string $parameterName
      * @param mixed $value
      * @return void
-     * @api
+     * @deprecated since Flow 7.0 use setHttpHeader or replaceHttpResponse instead. See also #linkToDocs
```

Comment:
`#linkToDocs`?  ;)
We should also mention here, that this only works with a `$componentClassName` of "SetHeaderComponent" or "ReplaceHttpResponseComponent"